cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmChangeUpdate.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.update;
+
+import java.util.Objects;
+
+import com.powsybl.iidm.network.Identifiable;
+
+/**
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class IidmChangeUpdate extends IidmChange {
+
+    public IidmChangeUpdate(Identifiable identifiable, String attribute, Object oldValue, Object newValue) {
+        super(identifiable);
+        this.attribute = Objects.requireNonNull(attribute);
+        this.oldValue = oldValue;
+        this.newValue = Objects.requireNonNull(newValue);
```

Comment:
Are you sure that newValue is never null? As oldValue can be null, I guess newValue could also

Reply:
Done